news/2022-09-01-how-zetera-became-a-medal-hunter.md

```diff
@@ -0,0 +1,55 @@
+> Besides that, there are people in this community that deserve my user title a lot more than myself, and I strongly hope that staff considers giving out more of my title to those who actually deserve it. Even if it's just through means of competition like it was done with The Girl in the Forest, it would give the title more relevance and prestige. Heck, I keep dropping out of the 90% medals club myself.
+
+So just to provide more context to his comment above, Zetera was a [GMT (Global Moderator)](/wiki/People/The_Team/Global_Moderation_Team) for 3 years, this team serves purpose to tackle community issues and player moderation. Moddingv2 is how the new beatmap modding system was called, it's a complete overhaul of the ancient beatmap modding system, where instead of having a forum thread for every map submitted (which got messy real quick), a more streamlined system was put in place to properly sort feedback and general praise, as well as feedback pertaining to a specific part of a beatmap with the brand new timeline. He also mentored new mappers via the Mentorship program (and will be returning to it again!), which is a program that puts people who are eager to learn about mapping with experiences mappers who are eager to share their knowledge. One thing he later mentioned as well is that he led the modern standard spotlight team behind the spotlight playlists for lazer for the first two seasons in 2020, these are the playlists that you will find in lazer and from which you can earn medals if you are one of the best!  
+
+You might ask yourself what Zetera meant when he said that he kept "dropping out of the 90% medals club". Well, in 2019 the medal hunter community created [Osekai](https://osekai.net/home/), a website made for alternative rankings, in-depth profile information, medal solutions etc. On their [Discord server](https://discord.com/invite/8qpNTs6) there are "clubs" that people "join" by receiving a Discord role based upon the number of medals they have. The club Zetera is referring to is the 90% medal club, which at the time of writing this currently consists of [only 44 people](https://osekai.net/rankings/?ranking=Medals&type=Users) in the entire game!
+
+If you are interested in medal hunting definitely look into Osekai and join the Medal Hunters Discord server, and definitely stay on the lookout for new medals as Ephemeral teased new [hush-hush](https://twitter.com/osekaiapp/status/1564610856381464580) the community found [some more medals](https://twitter.com/osekaiapp/status/1564871086617071616) by guessing the links to them!
```

Comment:
dunno if the last part should be in the article if they haven't been released or teased yet (@Ephemeralis)

```suggestion
If you are interested in medal hunting definitely look into Osekai and join the [Medal Hunters Discord server](https://discord.com/invite/8qpNTs6), and definitely stay on the lookout for new medals, especially since Ephemeral has just teased [a new Hush-Hush medal](https://twitter.com/osekaiapp/status/1564610856381464580) and with the community discovering [even more upcoming medals](https://twitter.com/osekaiapp/status/1564871086617071616) by finding their assets on the website.
```

Reply:
i mean it was tweeted publicly so I guess it'd be a open secret?

Reply:
doesn't mean it should be on a news post. just mentioning this because there was a [pr](https://github.com/ppy/osu-wiki/pull/5736) that was going to add hush-hush medals and their known unlock requirements to the wiki, but they had to be removed even though there is a whole community that knows this stuff, which in itself is fine

Reply:
i'd avoid mentioning the fact hint solutions are available there outright, though phrasing it as a community of people interested in medals should be fine

Reply:
Wala's concerns were addressed by [Eph ](https://discord.com/channels/188630481301012481/218677502141399041/1016089780842406031)in the Discord, as for what Eph said it has been applied in [bc9d0cc](https://github.com/ppy/osu-wiki/pull/7949/commits/bc9d0cc8445cc93145a4ab04ce28e99991fe7269)